support/monitoring/README.md

```diff
@@ -174,7 +174,7 @@ For creating a dashboard to monitor the cluster:
 
 Click '+' button on left panel and select ‘Import’.
 
-Enter 3119 dashboard id under Grafana.com Dashboard.
+Copy paste the json file in grafana-dashboards folder. 
```

Comment:
We should specify exactly which json file.

Reply:
Can you also add a quick description on what loading this json does (i.e., loading a predefined dashboard) and what this particular dashboard contains. an example screenshot of the dashaboard would be pretty good here.

Reply:
OK Gonna do it later today

Reply:
Sorry, I forgot this PR. I have added a sentence to do this so.. Please review again.